recipe/meta.yaml

```diff
@@ -108,10 +111,10 @@ outputs:
         # pytorch-cpu 1.1 which we built before conda-forge had GPU infrastructure
         # built into place.
         # https://github.com/conda-forge/pytorch-cpu-feedstock/issues/65
-        - pytorch-cpu = {{ version }}  # [cuda_compiler_version == "None"]
-        - pytorch-gpu = 99999999       # [cuda_compiler_version == "None"]
-        - pytorch-gpu = {{ version }}  # [cuda_compiler_version != "None"]
-        - pytorch-cpu = 99999999       # [cuda_compiler_version != "None"]
+        - pytorch-cpu =={{ version }}  # [cuda_compiler_version == "None"]
+        - pytorch-gpu ==99999999       # [cuda_compiler_version == "None"]
+        - pytorch-gpu =={{ version }}  # [cuda_compiler_version != "None"]
+        - pytorch-cpu ==99999999       # [cuda_compiler_version != "None"]
```

Comment:
@beckermr do you think this is "ok". I don't want to change the pinning structure too much if we can avoid it.